### PR TITLE
Add other perf node

### PR DESCRIPTION
**Pull request by achamayou**

### CODE REVIEW

File: .azure-pipelines-templates/matrix.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ parameters:
     san:
       cmake_args: '-DSAN=ON'
     perf:
-      cmake_args: '-DPERF=ON'
+      cmake_args: '-DISTRIBUTE_PERF_TESTS=`../.nodes.sh`'
```
Comment (jumaffre) [inline .azure-pipelines-templates/matrix.yml]: Not sure where to write this: now that we've got distributed machines, the forwarding test with signatures can be run with a client for each backup  (`-cn` option).

File: .nodes.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+HN=$(hostname)
+
+if [ "$HN" == "ccf-ci-5" ]; then
+    echo "-n 10.0.0.6 -n 10.0.0.8 -n 10.0.0.11"
```
Comment (eddyashton) [inline .nodes.sh]: Can we use hostnames here? ie `-n ccf-ci-6 -n ccf-ci-7 -n ccf-ci-8`?

File: .azure-pipelines-templates/matrix.yml (hunk 1 above)
Comment (achamayou) [reply .azure-pipelines-templates/matrix.yml]: Please open an issue.

File: .nodes.sh (hunk 2 above)
Comment (achamayou) [reply .nodes.sh]: No, they resolve to public IPs.

**Comment by cimetrics** [comment]
add_other_perf_node@[4784 aka 20200211.14](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4784&view=results) vs master ewma over 30 builds from [4442](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4442&view=results) to [4762](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4762&view=results)
images

**Pull request merged**